medusa/medusacli.py

```diff
@@ -332,12 +332,15 @@ def build_index(medusa_config, noop):
 
 
 @cli.command(name='purge')
+@click.option('--prefer-incomplete', default=False, is_flag=True,
+              help='Prioritise incomplete backups when purging by count')
```

Comment:
**Question:** in which case do you think we may prefer to keep incomplete backups? They cannot be restored anyway so it doesn't seem like something we'd want to keep.